enterprise/internal/oobmigration/migrations/codeintel/migrator.go

```diff
@@ -62,6 +64,11 @@ type migratorOptions struct {
 	// batchSize limits the number of rows that will be scanned on each call to Up/Down.
 	batchSize int
 
+	// numRoutines is the maximum number of routines that can run at once on invocation of the
+	// mgirator's Up or Down methods. If zero, a number of routines proportional to available
```

Comment:
```suggestion
	// migrator's Up or Down methods. If zero, a number of routines proportional to available
```

Why does this say "proportional" instead of "equal"?

Reply:
```suggestion
	// migrator's Up or Down methods. If zero, a number of routines equal to the number of available
```